Frameworks/UIKit/UIScrollView.mm

```diff
@@ -169,87 +162,29 @@ - (id)_dManipGestureCallback:(_UIDMPanGestureRecognizer*)gesture {
 
 - (void)_initUIScrollView {
     // Store a strongly-typed backing scrollviewer
-    _scrollViewer = rt_dynamic_cast<WXCScrollViewer>([self xamlElement]);
-    if (!_scrollViewer) {
+    WXCGrid* scrollViewGrid = rt_dynamic_cast<WXCGrid>([self xamlElement]);
+    WXCScrollViewer* scrollViewer = nil;
```

Comment:
I just wondered if you _scrollViewer = XamlControls::GetScrollViewer(scrollViewGrid) and avoid the local variable (sort of like what you did with _contentCanvas below)

---
In reply to: [106552317](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106552317) [](ancestors = 106552317,106549343)